app/editor/src/features/admin/products/constants/subscriberColumns.tsx

```diff
@@ -91,10 +91,14 @@ export const subscriberColumns = (
           )}
           disabled={true}
           title={
-            product.subscribers.filter((u) => u.id === cell.original.id)[0]
```

Comment:
Why going through the array twice?  Can't you just do `product.subscribers.filter((u) => u.id === cell.original.id)?.requestedIsSubscribedStatus ? 'Request is to SUBSCRIBE' : 'Request is to UNSUBSCRIBE'`?